client/components/edit/datastream/DatastreamModal.tsx

```diff
@@ -23,15 +24,17 @@ const DatastreamModalContent = ({ datastreamModalState }: { datastreamModalState
 
 const DatastreamModal = (): React.ReactElement => {
     const {
-        state: { datastreamModalState, isDatastreamModalOpen },
-        action: { toggleDatastreamModal },
+        state: { datastreamModalState },
     } = useEditorContext();
+    const {
+        action: { closeModal, isModalOpen },
+    } = useGlobalContext();
 
     return (
         <Dialog
             className="datastreamModal"
-            open={isDatastreamModalOpen}
-            onClose={toggleDatastreamModal}
+            open={isModalOpen("datastream")}
+            onClose={() => closeModal("datastream")}
```

Comment:
In other components, we define a local `closeXModal` function instead of using the `() => ...` syntax. Both approaches are valid, but I wonder if we should choose one or the other for consistency. I don't feel a lot of pressure to change anything, but let me know if you have an opinion!

Reply:
I think we can leave this alone for now.